Data/HashMap/Base.hs

```diff
@@ -461,12 +461,12 @@ lookup :: (Eq k, Hashable k) => k -> HashMap k v -> Maybe v
 -- unboxed sums automatically. That seems likely to happen at some
 -- point (possibly as early as GHC 8.6) but for now we do it manually.
 lookup k m = case lookup# k m of
-  (# (# #) | #) -> Nothing
+  (# _ | #) -> Nothing
```

Comment:
Does this have any impact on performance? I would hope not, but if it does would it be worth it to condition this on GHC versions that support nullary tuples? It would clutter the code quite a bit more though :/

Reply:
It turns out they all support nullary tuples; I just got confused.